cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/etl/transform/ScriptFilterTransform.java

```diff
@@ -102,7 +102,7 @@ public void transform(StructuredRecord input, Emitter<StructuredRecord> emitter)
         metrics.count("filtered", 1);
       }
     } catch (Exception e) {
-      throw new IllegalArgumentException("Invalid filter condition.", e);
+      throw new IllegalArgumentException("Could not transform input: " + input, e);
```

Comment:
We might need to be careful as logging with user data might not be warned if it contains sensitive data.